chromeos.kernelci.org

```diff
@@ -0,0 +1,224 @@
+#!/bin/sh
+
+# Periodic job run on staging.kernelci.org to merge together all the pending
+# PRs and update the staging.kernelci.org branches, then update Docker images
+# and trigger a full build/test cycle with a kernel branch based on linux-next.
+
+set -e
+
+CHROMEOS_SETTINGS="data/chromeos.ini"
+
+cmd_pull() {
+    echo "Updating local repository"
+    git pull --ff-only
+}
+
+cmd_jenkins() {
+    echo "Updating Jenkins jobs"
+    ./pending.py kernelci-jenkins --settings=${CHROMEOS_SETTINGS} --main=chromeos.kernelci.org --push
+
+    [ -d checkout/kernelci-jenkins/data ] || {
+        # To set this up:
+        # git clone https://github.com/kernelci/kernelci-jenkins-data.git data
+        # cd data; git-crypt unlock
+        # cd ..; ln -s data/bot.staging.kernelci.org/env .env
+
+        echo "Encrypted kernelci-jenkins-data repository not set up"
+        return 1
+    }
+
+    echo "Updating encrypted data checkout"
+    topdir=$(pwd)
+    cd checkout/kernelci-jenkins/data
+    git remote update origin
+    data_diff=$(git diff origin/main)
+    if [ -n "$data_diff" ]; then
+        git checkout origin/main
+        cd "$topdir/checkout/kernelci-jenkins"
+        echo "Recreating Jenkins container"
+        docker-compose down
+        docker-compose up --build -d
+        sleep 60  # magic - another way to do this would be to poll something
+    else
+        cd "$topdir/checkout/kernelci-jenkins"
+        # ToDo: restart the container but only when there were some changes in
+        # kernelci-jenkins to avoid aborting running jobs unnecessarily
+    fi
+    cd "$topdir"
+
```

Comment:
Please drop this, it's the same Jenkins instance as staging.kernelci.org.

Reply:
I think that issue description should be changed then. At the moment it says that there should be a `chromeos.kernelci.org` branch created using pending PRs also for `kernelci-jenkins`

Reply:
We need to create the branch with pending PRs but no need to recreate the Docker container.  The jobs will be picking up the branch created by the script.

Reply:
Got it.